shared-libs/validation/package-lock.json

```diff
@@ -31,7 +31,124 @@
       }
     },
     "@medic/message-utils": {
-      "version": "file:../message-utils"
```

Comment:
You will need to leave this one as it is:
```
"@medic/message-utils": {
    "version": "file:../message-utils"
}
```
I think it'll give you error when trying to install the project with `npm ci` (which bases the install on the package-lock.json)
I've sent you some info in slack but pls reach out if you have questions :)

Reply:
cools - thanks very much for the feedback. So it sounds like I need to:
1. not run `npm install` after I update with `updates --minor` ?
2. for `shared-libs/validation/package-lock.json` revert back to the [21 lines](https://github.com/medic/cht-core/blob/master/shared-libs/validation/package.json) that are in `master` now?

I'll revisit my automation to see what I'm doing wrong :sweat_smile:

Reply:
hopefully fixed in db2fe7844